try-it-out/docs/pages/deploy-devops-tools.rst

```diff
@@ -0,0 +1,63 @@
+.. _Deploy Devops Tools:
+-----------------------
+Deploy The Devops Tools
+-----------------------
+
+In this section, you will be deploying all of the DevOps tools that we will be using within this tutorial. This consists of deploying a Jenkins instance, a SonarQube server, and a local (insecure) Docker registry. 
```

Comment:
What do you think of this wording?
 “In this section, you will be deploying all of the DevOps tools that we will be using within this tutorial: Jenkins, SonarQube, and a local (insecure) Docker registry.